articles/container-registry/container-registry-faq.md

```diff
@@ -441,3 +441,83 @@ This setting also applies to the `az acr run` command.
+
+## DOCKER_COMMAND_ERROR
+
+This error means that docker client for CLI could not be found, which precludes finding docker version, evaluating docker daemon status and ensuring docker pull command can be ran.
+
+*Potential solutions*: installing docker client; adding docker path to the system variables.
+
+## DOCKER_DAEMON_ERROR
+
+This error means that the docker daemon status is unavailable, or that it could not be reached using the CLI. This means that docker operations (e.g., login, pull) will be unavailable through the CLI.
+
+*Potential solutions*: Restart docker daemon, or validate that it is properly installed.
+
+## DOCKER_VERSION_ERROR
+
+This error means that CLI was not able to run the command `docker --version`.
+
+*Potential solutions*: try running the command manually, make sure you have the latest CLI version, and investigate the error message.
+
+## DOCKER_PULL_ERROR
+
+This error means that the CLI was not able to pull a sample image to your environment.
+
+*Potential solutions*: validate that all components necessary to pull an image are running properly.
+
+## HELM_COMMAND_ERROR
+
+This error means that helm client could not be found by the CLI, which precludes other helm operations.
+
+*Potential solutions*: verify that helm client is installed, and that its path is added to the system environment variables.
+
+## HELM_VERSION_ERROR
+
+This error means that the CLI was unable to determine the Helm version installed. This can happen if the Azure CLI version (or if the helm version) being used is obsoleted.
+
+*Potential solutions*: update to the latest Azure CLI version or to the recommended helm version; run the command manually and investigate the error message.
+
+## CONNECTIVITY_DNS_ERROR
+
+This error means that the DNS for the given registry login server was pinged but did not respond to it, which means it is unavailable. This can indicate some connectivity issues. This can also mean that the registry does not exist, that the user does not have the permissions on the registry (to retrieve its login server properly), or that the target registry is in a different cloud than the one being used in the Azure CLI.
+
+*Potential solutions*: validate connectivity; verify spelling of the registry, and that registry exists; verify that the user has the right permissions on it and that the registry's cloud is the same that is being used on Azure CLI.
+
+## CONNECTIVITY_FORBIDDEN_ERROR
+
+This means that the challenge endpoint for the given registry responded with a 403 Forbidden HTTP status. This means that user don't have access to the registry, most likely due to a VNET connection.
+
+*Potential solutions*: remove VNET rules or add the current client IP to the allowed list.
+
+## CONNECTIVITY_CHALLENGE_ERROR
+
+This error means that the challenge endpoint of the target registry did not issue a challenge.
+
+*Potential solutions*: try again after some time. If error persists, contact the Azure Container Registry team.
+
```

Comment:
Is there a documented way to contact the ACR team?

Reply:
We actually have acrsup@microsoft.com as a reference in github.com/azure/acr issue template. I would recommend you ask customers to open a github issue.

Reply:
I see. So how should I point to that? Something like
"please open a github issue to the Azure Container Registry team."
or "please open an issue according to [here](https://github.com/Azure/acr/blob/master/.github/ISSUE_TEMPLATE/bug_report.md)." ?

Reply:
https://aka.ms/acr/issues